pkg/datapath/linux/config/config.go

```diff
@@ -720,6 +730,17 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 	}
 	cDefinesMap["EPHEMERAL_MIN"] = fmt.Sprintf("%d", ephemeralMin)
 
+	for _, fn := range h.nodeExtraDefines {
+		defines, err := fn()
+		if err != nil {
+			return err
+		}
+
+		for key, value := range defines {
+			cDefinesMap[key] = value
```

Comment:
Can we return an error here if `key` already exists? It probably doesn't make sense to allow overwriting defines.